Example/PushDecryptionService/NotificationService.swift

```diff
@@ -13,7 +13,7 @@ class NotificationService: UNNotificationServiceExtension {
         bestAttemptContent = (request.content.mutableCopy() as? UNMutableNotificationContent)
         if let bestAttemptContent = bestAttemptContent {
             let topic = bestAttemptContent.userInfo["topic"] as! String
-            let ciphertext = bestAttemptContent.userInfo["ciphertext"] as! String
+            let ciphertext = bestAttemptContent.userInfo["blob"] as! String
```

Comment:
push structure changed?

Reply:
yes Harry did not follow the payload from issue, and I thought it will be faster to change it on our end then wait